contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+Cohort retention tables are also useful for tracking revenue retention, especially if you charge based on usage. Positive revenue retention (i.e. more than 100%) indicates users increase their spend over time. Needless to say, this is a good thing.
+
+> 😱 **Pedantic Sidebar:** For those thinking "if retention and churn rate are just the inverse of each other, why can't I create a churn rate table instead?" My answer is this... yes, you could, but it would be a dumb thing to do. Why? Because retention tables make more intuitive sense. Be a rebel if you like, but you're dead to me. 
+
+## How to improve retention and reduce churn
+
+"Let's reduce churn" isn't quite akin to "nobody ever got fired for buying IBM", but it's worth remembering why reducing churn is desirable:
+
+1. High churn is a likely sign you don't have product-market fit.
+
+2. High churn leads to increased sales and/or marketing spend to compensate.
+
+3. Reducing churn compounds – i.e. revenue **and** marketing ROI increase.
+
+So, how do you improve retention and reduce churn? The glib answer is "make product better", but we can do a bit better than that:
+
+- **Check for involuntary churn:** Involuntary churn typically means billing problems – e.g. expired cards, out-of-date info etc. In e-commerce, abandoned baskets could also be an involuntary churn problem.
+
+- **Improve new user activation:** New user activation is an [important B2B product metric](/blog/b2b-saas-product-metrics). Use funnel analysis and session recordings to identify problems in your onboarding flow. Sometimes small tweaks to user flows can have a big impact.
+
+- **Talk to users:** Stop me if you've heard this before, but talking to users is vital in any context – not just when you're trying to solve a retention problem. We're constantly talking to users to understand what they need, and problems we can solve. We built the [PostHog User Interview](https://github.com/posthog/user-interview-app) and [Feedback apps](https://github.com/PostHog/feedback-app) to help us do se. We document our [user feedback process](/handbook/product/user-feedback) in our public handbook.
+
+- **Consider marketing and product alignment:** Sometimes poor retention comes from acquiring the wrong users. This could be a marketing problem, or you might have the wrong ideal customer profile.
+
+- **Learn about your power users:** It's great to have power users, but they're also more sensitive to product changes. It's important to identify who they are, what they do, and what they value so you can avoid retention problems. See: [How to identify and analyze power users in PostHog](/tutorials/power-users).
+
+- **Build great new features:** If you're talking to your users, and learning about your power users, then you should be well-placed to build awesome new features your existing (and potential) customers will love. I'm told this is "creating value" in the official business lingo.
```

Comment:
"The glib answer is "make product better", but we can do a bit better than that:"

*proceeds to be glib*